pkg/trace/api/container.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+//go:build !linux
+// +build !linux
+
+package api
+
+import (
+	"context"
+	"net"
+	"net/http"
+)
+
+func connContext(ctx context.Context, c net.Conn) context.Context {
+	return ctx
```

Comment:
I would leave a comment here, and for GetContainerID, that says something like `// unimplemented` so it's clear that this is just the default non-linux case. It confused me when I first looked at it.